exercises/robot-name/.meta/exercise-data.yaml

```diff
@@ -0,0 +1,63 @@
+exercise: RobotName
+package_comment: "# This will be class 'RobotName', defined using Moo (https://perlmaven.com/oop-with-moo)"
+lib_comment: '# Find modules in the same dir as this file.'
+plan_comment: '# This is how many tests we expect to run.'
+
+moo: true
+methods: new name reset_name
+
+plan: 7
+  # plan includes can_ok of `methods` plus the tests below.
+
+# Tests: inline here, since there is no canonical-data.json for this exercise
+tests: |-
+  my $robot = RobotName->new;
+  isa_ok $robot, 'RobotName';
+
+  my $name  = $robot->name;
+  ok $robot->name =~ /[A-Z]{2}[0-9]{3}/, 'Name should match schema';
+  is $name, $robot->name, 'Name should be persistent';
+  ok $robot->name ne RobotName->new->name,
+    'Robots should have different names';
+  ok $robot->reset_name ne $name,
+    'reset_name should change the robot name';
+  ok $robot->name, 'reset_name should not leave the name empty';
```

Comment:
Since the generator gives us `Test2::V0`, we'll want to replace these `ok` tests with `is` ones. This should give us clearer test output and stop warnings about uninitialized values for unsolved tests.